mongodbatlas/version.go

```diff
@@ -0,0 +1,15 @@
+package mongodbatlas
+
+import "fmt"
+
+const (
+	// UserAgent the default user agent string reported by this client
+	UserAgent = "go-mongodbatlas"
+	// ClientVersion the version of the current API client
+	ClientVersion = "0.4.0" // Should be set to the next version planned to be released
+)
+
+// DefaultUserAgent constructs the default user agent
+func DefaultUserAgent() string {
```

Comment:
[nit] similar to the const I don't see a value on exporting this or even why we have 
`UserAgent` as a const and `DefaultUserAgent` as a func
we should just offer one, maybe 

```go
const libraryName = "go-mongodbatlas"
const Version = "0.4"
var userAgent = fmt.Sprintf("%s/%s (%s;%s)", libraryName, Version, runtime.GOOS, runtime.GOARCH)
```

Reply:
nice!